### ISSUE TITLE
Not clicking?

### ISSUE THREAD

**Issue by raven7931** [d4rken/gplay-batchtool#7]
Not clicking the Library Tab, or the remove button, is there a fixed resolution? I am trying to run this on an emulator.

**Comment by turbogauchiste** [comment]
> 
> 
> Not clicking the Library Tab, or the remove button, is there a fixed resolution? I am trying to run this on an emulator.

Same issue here (not on an emulator though), but you may temporarily downgrade your Google Play app: in the gplay app details in your phone settings, there may be three dots on the top right and an option to be clicked "uninstall upgrades" or something like that. It will restore the factory version of the app, one most likely to be compatible with gplay-batchtool. 
Hope this helps!

**Comment by Coded-Message** [comment]
> Not clicking the Library Tab, or the remove button, is there a fixed resolution? I am trying to run this on an emulator.

I was able to get around this by running the tool then clicking the profile pic in the top right > my apps & games > library. It should run normally now :)

**Comment by aparedero** [comment]
@ChaoticHub you're right. With latest changes in GPlay Store, new location of buttons makes this app doesn't find the path, but if you go through library manually, application will be able to delete list.
